pkg/network/render.go

```diff
@@ -293,6 +294,17 @@ func IsChangeSafe(prev, next *operv1.NetworkSpec, infraStatus *bootstrap.InfraSt
 	// Check kube-proxy
 	errs = append(errs, isKubeProxyChangeSafe(prev, next)...)
 
+	// Verify ipForwarding. This is necessary because the API was merged without any parameter validation, and it is
+	// not allowed to narrow the list of allowed values for an API field after it was released.
+	// For more details, see https://github.com/openshift/cluster-network-operator/pull/2127 and
+	// https://github.com/openshift/api/pull/1691.
+	if mustValidateIPForwarding(prev, next) {
```

Comment:
This check is OVNKubernetes specific, if we decide to keep it as a change check it should be placed somewhere [isOVNKubernetesChangeSafe](https://github.com/openshift/cluster-network-operator/blob/c45e5c115de6a1917a898222381f0d3dec1c949c/pkg/network/ovn_kubernetes.go#L988C6-L988C31)

Reply:
I removed that commit